src/utils/index.ts

```diff
@@ -93,7 +93,7 @@ export const getIntervalInSeconds = ({
     timeUnit,
     interval,
 }: {
-    timeUnit?: string | number;
+    timeUnit?: string | number | null;
```

Comment:
<img width="556" alt="image" src="https://github.com/binary-com/SmartCharts/assets/56330681/9c8aeebb-f2a1-4ffb-aead-dd81384e92b8">

Can we add a null check here since we are adding null to the `timeUnit` type?

Reply:
Done now thanks